queue.c

```diff
@@ -166,7 +166,9 @@ typedef xQUEUE Queue_t;
  * The pcQueueName member of a structure being NULL is indicative of the
  * array position being vacant. */
     PRIVILEGED_DATA QueueRegistryItem_t xQueueRegistry[ configQUEUE_REGISTRY_SIZE ];
-
+    /* MISRA Ref 8.4.2 [Declaration shall be visible] */
```

Comment:
The annotation should be placed just above the declaration not below.

Reply:
updated.